update 0.01 version of doc (#8)
just-foss/community: README.md

# community

> # what is the just-foss organisation?

Reviewer: ```suggestion
# what is the just-foss organization?
```

The aim of just-foss organisation it to be a free, opensource, remote, regular meeting, written by and for hackers in computer science. 

> We want to provide a community which is not driven by status, but rather by content and real humanity. ( we like humor but not hyped code of conducts, ok? :) ) 

Reviewer 2: I would be in favor of adopting a CoC, personally - trying to fight that is probably going to come back and bite everyone eventually. I don't think that has to be opposed to humor or humanity, hehe.

Author: ok done thx. PR created

During the meetings we talk about topics that might be interesting for others in different domains.

The aim of just foss is to be indipendent content from any organisation, programming languages, stereotypes.

Allthough, the just-foss community will favor functional code and principles over imperative/OO oriented languages.

> # Which type of talks are accepted:

Reviewer: ```suggestion
# Which type of talks are wanted
```

A talk should be at least something NOT commercial/business driven. It should aim to be honest, and driven by quality, for sharing with people.

It can be a pure theoretical one, e.g Queue theories, or can be mixed by a theory and pragmatic approach.

For example some domains:  #ai #networking #linux #functional programming #software-design #architecure #security #performance #monitoring etc.

The main goal of just-foss, would aim to share valid knowledge for people to different domains.

# How look like a meeting:

We will do a remote meeting when we have at least 4 talks. Each talk should from 20/40 minutes  duration, with slides.

> The meeting will happen over a gotomeeting invitaition, and it will be registered for sharing on the web. 

Reviewer: ```suggestion
The meeting will happen via a GoToMeeting invitation, and it will be registered for sharing on the web.
```

We will do preferrably meetings with camera, but if people don't want to show up for some reason, we don't require it, allthough showing up create more connections feeling :)

After the presentation, attendees can raise question, if not they can still get in contact via Issues etc with the presenter, if they have later question.

> # How can I bee part of the movement?

Reviewer: ```suggestion
# How can I be part of the movement?
```

> * If you want to present something, open an issue here and describe what you want to talk about, how long it should be, and why you think your content is something usefull to share.

Reviewer: ```suggestion
* If you want to present something, open an issue here and describe what you want to talk about, how long it should be, and why you think your content is something useful to share.
```

> * If you want to just listen, have a look at the twitter #just-foss tag, or the other issue in GitHub. We shouldn't have problems with different attendes.

Reviewer: I think tags with dashes are not supported by twitter ;)

Reviewer: I think hash tags with minus are still not supported. 🤭

* If you want to be a maintainer of #just-foss, feel free to write us why you want to be part of the organisation.
  We are searching for new maintainers and welcome diversity, allthough beeing a maintainer isn't necessary for listen or present.

# Questions?

Open an issue on GitHub `just-foss/community` for any question.

# Workflow notes:

> Our process is versioned to `0.0.1` version we can improve it over time and we will do it.

Reviewer: ```suggestion
Version: v0.1.0
```
:question:

# community

# what is the just-foss organization?

> The aim of just-foss organisation it to be a free, opensource, remote, regular meeting, written by and for hackers in computer science. 

Reviewer: To which one do we stick? Organization vs organisation?

Author: organization probably? what do you prefer

We want to provide a community which is not driven by status, but rather by content and real humanity.

During the meetings we talk about topics that might be interesting for others in different domains.

> The aim of just foss is to be indipendent content from any organisation, programming languages, stereotypes.

Reviewer: just-foss ? We should stick to one approach.

Reviewer: Typo: indipendent

> Allthough, the just-foss community will favor functional code and principles over imperative/OO oriented languages.

Reviewer: Typo: Although

Reviewer: What means imperative/00?

Author: imperative style, Object oriented

Author: but I can remove it..

# Which type of talks are wanted

A talk should be at least something NOT commercial/business driven. It should aim to be honest, and driven by quality, for sharing with people.

It can be a pure theoretical one, e.g Queue theories, or can be mixed by a theory and pragmatic approach.

For example some domains:  #ai #networking #linux #functional programming #software-design #architecure #security #performance #monitoring etc.

The main goal of just-foss, would aim to share valid knowledge for people to different domains.

> # How look like a meeting:

Reviewer: I guess this should mean `How a meeting looks like`, right?

We will do a remote meeting when we have at least 4 talks. Each talk should from 20/40 minutes  duration, with slides.

The meeting will happen via a GoToMeeting invitation, and it will be registered for sharing on the web.

We will do preferrably meetings with camera, but if people don't want to show up for some reason, we don't require it, allthough showing up create more connections feeling :)

After the presentation, attendees can raise question, if not they can still get in contact via Issues etc with the presenter, if they have later question.

# How can I be part of the movement?

* If you want to present something, open an issue here and describe what you want to talk about, how long it should be, and why you think your content is something useful to share.

* If you want to just listen, have a look at the twitter #just-foss tag, or the other issue in GitHub. We shouldn't have problems with different attendes.

* If you want to be a maintainer of #justfoss, feel free to write us why you want to be part of the organisation.
  We are searching for new maintainers and welcome diversity, allthough beeing a maintainer isn't necessary for listen or present.

# Code of conduct 

> We use code of conduct, see LINK[FIXME]

Author: this should be fixed by next pr
